### DISCUSSION TITLE
V1.3.0

### DISCUSSION THREAD

**Discussion by TimoteoMendes** [danak6jq/Seeed-LoRa-E5#21] [General]
Hello,

Thanks for sharing your work!
I am able to use it in a costume board I have using the same lora-e5.
But because of the new updates it runs fine in the stm32cube fw_wl V1.2.0 and cubeMX 6.6.1 but the latest releases when migrate the code for the new stm32cube fw_wl V1.3.0 and cubeMX 6.8.1 it does not work anymore.
Can you give me a clue where should I change it?
Thanks.

Tim

**Comment by danak6jq** [comment]
Did you resolve this issue? I will have the opportunity soon to update to v1.3 if you haven't already. If you have, I invite you to open a PR to merge the update

**Comment by TimoteoMendes** [comment]
Hello Dana,
Yes it is already working.
I am new in this git world if you explain me how to do it I can happily
share it!
Thanks,
Tim

A sábado, 6/05/2023, 23:05, Dana Myers ***@***.***> escreveu:

**Comment by TimoteoMendes** [comment]
Hello Dana,

I have been testing this but seens to have some problem when I generate a
new projecto from IOC.
If I do so without any changes, the project is created but when compile it
complains about the BSP drivers, I copy then to the new project and add it
in paths and symbols, it compile with no errors and it starts ok and can
register but does not continue send messages every 10s...
Do I need to change some thing else?
In fact I would like to use the on_event option as I just want to send new
messages when something happens.
Can you give a hand?

Thanks.
Tim

PS- your code changed to V1.3.0 and latest MX in attach

it was not possible to send it.
[image: image.png]

**Comment by TimoteoMendes** [comment]
Hello Dana,

I could figure it out.
You have a lot of code inside the lora_app.
Now I can make it to work properly.

I found that you are using the LSI and not the external tcx?
In the seeed wiki it refers to have a 32MHz Cristal. Is it correct?

Thanks,
Tim

A segunda, 8/05/2023, 12:11, Timóteo Mendes ***@***.***>
escreveu:

**Reply by danak6jq** [reply]
This is a port of the ST example application. I've already ported my RAK3172 repo to 1.3, I'll have a look at this. I don't recall it was a major change. There *are* changes to the ST example application in 1.3 as ST added LMHandler interfaces.

**Comment by danak6jq** [comment]
I've created a FW1.3 branch with a pull request which has the initial FW 1.3 changes - literally, I told STM32CubeIDE to update from 1.2 to 1.3. I haven't tested this yet but the changes look reasonable. You might try this to see if it helps move to 1.3.

It may require an additional change or two to lora_app.c - I've done this update in a private repo and will check.
